provider/kv/kv_config.go

```diff
@@ -369,6 +370,39 @@ func (p *Provider) getTLSSection(prefix string) []*tls.Configuration {
 	return tlsSection
 }
 
+// GetTLSClientCert create tls client header configuration from labels
```

Comment:
`// getTLSClientCert creates TLS client header configuration from labels`
instead of 
`// GetTLSClientCert create tls client header configuration from labels`